frontends/mit-open/src/pages/UnitsListingPage/UnitsListingPage.tsx

```diff
@@ -0,0 +1,408 @@
+})
+
+const UnitTitle = styled(Typography)(({ theme }) => ({
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.h4,
+}))
+
+const UnitDescriptionContainer = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "flex-start",
+  paddingBottom: "8px",
+})
+
+const UnitDescription = styled(Typography)(({ theme }) => ({
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.body2,
+}))
+
+const AcademicIcon = styled(RiBookOpenLine)({
+  width: "32px",
+  height: "32px",
+})
+
+const ProfessionalIcon = styled(RiSuitcaseLine)({
+  width: "32px",
+  height: "32px",
+})
+
+const GridContainer = styled(Box)(({ theme }) => ({
+  display: "grid",
+  gap: "25px",
+  gridTemplateColumns: "repeat(2, 1fr)",
+  [theme.breakpoints.down("md")]: {
+    gridTemplateColumns: "1fr",
+  },
+}))
+
+const UnitCard = styled(Card)({
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+})
+
+const UnitCardContent = styled(CardContent)({
+  display: "flex",
+  flexDirection: "column",
+  flexGrow: 1,
+})
+
+const LogoContainer = styled.div({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  height: "128px",
+})
+
+const UnitLogo = styled.img({
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "center",
+  alignItems: "center",
+  height: "50px",
+  maxWidth: "100%",
+})
+
+const ValuePropContainer = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "flex-start",
+  justifyContent: "flex-start",
+  flexGrow: 1,
+  paddingBottom: "16px",
+})
+
+const ValuePropText = styled(Typography)(({ theme }) => ({
+  alignSelf: "stretch",
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.body2,
+}))
+
+const CountsTextContainer = styled.div({
+  display: "flex",
+  justifyContent: "flex-end",
+  gap: "10px",
+})
+
+const CountsText = styled(Typography)(({ theme }) => ({
+  textAlign: "center",
+  color: theme.custom.colors.silverGrayDark,
+  ...theme.typography.body3,
+}))
+
+const unitLogos = {
+  [OfferedByEnum.Mitx]: "/static/images/unit_logos/mitx.svg",
+  [OfferedByEnum.Ocw]: "/static/images/unit_logos/ocw.svg",
+  [OfferedByEnum.Bootcamps]: "/static/images/unit_logos/bootcamps.svg",
+  [OfferedByEnum.Xpro]: "/static/images/unit_logos/xpro.svg",
+  [OfferedByEnum.Mitpe]: "/static/images/unit_logos/mitpe.svg",
+  [OfferedByEnum.See]: "/static/images/unit_logos/see.svg",
+}
+
+interface UnitSectionProps {
+  id: string
+  icon: React.ReactNode
+  title: string
+  description: string
+  units: LearningResourceOfferorDetail[] | undefined
+  courseCounts: Record<string, number>
+  programCounts: Record<string, number>
+  isLoading?: boolean
+}
+
+const UnitSection: React.FC<UnitSectionProps> = (props) => {
+  const {
+    id,
+    icon,
+    title,
+    description,
+    units,
+    courseCounts,
+    programCounts,
+    isLoading,
+  } = props
+  return (
+    <UnitContainer data-testid={`UnitSection-${id}`}>
+      <Box>
```

Comment:
I'm fairly certain that `Box` is just a div with MUI's `sx`, etc props available. So if you're not customizing it, could probably just use a `div`.